### PR TITLE
Add psfgen

### PR DESCRIPTION
**Pull request by Eigenstate**
Initial attempt at adding the Python interface to PSFGen, which prepares proteins and other molecules for molecular dynamics simulation. It's a simple C extension with some Python bindings. The test cases pass with Python 2.7 and 3.6 on my Linux machine.

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
I think that should be in ``bioconda``

File: recipes/psfgen/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+build:
+    number: 0
+    skip: true  # [win]
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+    host:
+        - python
```
Comment (marcelotrevisani) [inline recipes/psfgen/meta.yaml]: Please, add ``setuptools``, it seems you need it as a dependency for your build system

**Comment by Eigenstate** [comment]
I've contributed other simulation software to conda-forge (vmd-python) without issue and there are other Python packages of this nature already present. Is there grounds for inclusion in conda-forge vs. other package repositories? I'd like to keep everything in the same place so dependency management is easier for my users.

File: recipes/psfgen/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+build:
+    number: 0
+    skip: true  # [win]
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - setuptools
```
Comment (marcelotrevisani) [inline recipes/psfgen/meta.yaml]: Change ``setuptools`` to host please

**Comment by marcelotrevisani** [comment]
> I've contributed other simulation software to conda-forge (vmd-python) without issue and there are other Python packages of this nature already present. Is there grounds for inclusion in conda-forge vs. other package repositories? I'd like to keep everything in the same place so dependency management is easier for my users.

It is not a problem to add it to ``conda-forge``, just that on ``bioconda`` you can find a large repository for ``chemistry``, ``biology``, ``bioinformatics`` and so on
And you can add bioconda and conda-forge channels to your conda environment, that is not a problem at all

**Comment by marcelotrevisani** [comment]
Do you compile against ``openblas`` ?

**Comment by Eigenstate** [comment]
No, it shouldn't need BLAS

**Comment by marcelotrevisani** [comment]
Usually, it is not a problem when you have a dependency of a downstream project which rely on ``numpy``

My guess is: I think there is something wrong with  ``vmd-python``

**Comment by marcelotrevisani** [comment]
Could you please review and merge this PR:
https://github.com/conda-forge/vmd-python-feedstock/pull/6

**Comment by chrisburr** [comment]
macOS is failing because `vmd-python` has [macOS builds disabled for Python 3](https://github.com/conda-forge/vmd-python-feedstock/blob/master/recipe/meta.yaml#L16).

@Eigenstate As you also maintain that recipe, is this intentional?

**Comment by Eigenstate** [comment]
This was a good incentive to get OSX + Python 3 builds to actually work for vmd-python! Hopefully this will pass now.

**Comment by Eigenstate** [comment]
@conda-forge-admin please rerender

**Comment by Eigenstate** [comment]
@conda-forge-admin, please re-render.

File: recipes/psfgen/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }} -v{{ version }}.tar.gz
+    url: https://github.com/Eigenstate/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 2
```
Comment (chrisburr) [inline recipes/psfgen/meta.yaml]: Until the recipe is first deployed the build number should be 0:
```suggestion
    number: 0
```

File: recipes/psfgen/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{% set name = "psfgen" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "ecf5a23805e5fb4841c352203225bd7715e28e9251be2684d1e4c977953b5cbd" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }} -v{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/psfgen/meta.yaml]: This line isn't needed
```suggestion
```

**Pull request merged**